libs/client/API.lua

```diff
@@ -472,6 +472,11 @@ function API:removeGuildMember(guild_id, user_id, query) -- Guild:kickUser
 	return self:request("DELETE", endpoint, nil, query)
 end
 
+function API:getGuildBan(guild_id, user_id) -- Guild:getBan
```

Comment:
This should be below `getGuildBans`, as it is on the Discord docs, and as it is generated by my docsdump file.